<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelChecker</class>
 <widget class="QMainWindow" name="LabelChecker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>780</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>label checker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1080</width>
     <height>720</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1,0,4">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>标准模板图像</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="TemplateGraphicView">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>图像详情列表</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="GraphicDetialList">
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>16</width>
           <height>16</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>待检图像：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="MainGraphicView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>140</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>140</height>
         </size>
        </property>
        <property name="title">
         <string>图像源</string>
        </property>
        <widget class="QPushButton" name="EditTemplateButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>20</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>修改模板配置</string>
         </property>
        </widget>
        <widget class="QPushButton" name="OpenTargetPhotoButton">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>60</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>打开待检图像</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>190</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="OpenTargetStreamButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>100</y>
           <width>100</width>
           <height>30</height>
          </rect>
         </property>
         <property name="text">
          <string>打开摄像头视频流</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>190</width>
           <height>30</height>
          </rect>
         </property>
        </widget>
        <widget class="QComboBox" name="TemplatesComboBox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>190</width>
           <height>30</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>参数调整区：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>同类标签相似度容许值(%)：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="ClassSimilaritySldider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="singleStep">
             <number>0</number>
            </property>
            <property name="value">
             <number>90</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="ClassSimilaritySpinBox">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>90</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>缺陷标签相似度容许值(%, 本参数应当小于上一个参数)：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="NotGoodSimilaritySlider">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>95</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="NotGoodSimilaritySpinBox">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>95</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>容许线性误差像素数(pixel)：</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="LinearErrorSlider">
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="LinearErrorSpinBox">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>缺陷所占最小面积(pixel):</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="DefectMinAreaSlider">
            <property name="value">
             <number>6</number>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="DefectMinAreaSpinBox">
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>并行线程数:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSlider" name="horizontalSlider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>85</width>
              <height>30</height>
             </size>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>结果输出配置：</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="1" column="0">
         <widget class="QCheckBox" name="ExportDiffCheckBox">
          <property name="text">
           <string>输出差异图(Debug)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="ExportPatternCheckBox">
          <property name="text">
           <string>输出标签样式(Debug)</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="ExportHighPreDiffCheckBox">
          <property name="text">
           <string>输出高精差异图(Debug)</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="ExportDefectsCheckBox">
          <property name="text">
           <string>单独输出缺陷结果</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="ExportMatchedTemplateCheckBox">
          <property name="text">
           <string>输出匹配后模板图(Debug)</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>标签比对进度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="CopareProgressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>324</x>
      <y>75</y>
      <width>118</width>
      <height>55</height>
     </rect>
    </property>
    <property name="title">
     <string>调试选项</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="geometry">
      <rect>
       <x>424</x>
       <y>90</y>
       <width>97</width>
       <height>72</height>
      </rect>
     </property>
     <property name="title">
      <string>日志等级设置</string>
     </property>
     <addaction name="actionDEBUG"/>
     <addaction name="actionINFO"/>
    </widget>
    <addaction name="menu_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionDEBUG">
   <property name="text">
    <string>DEBUG</string>
   </property>
  </action>
  <action name="actionINFO">
   <property name="text">
    <string>INFO</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
